src/main/java/com/minecolonies/blocks/BlockHutFarmer.java

```diff
@@ -13,6 +25,30 @@ protected BlockHutFarmer()
     }
 
     @Override
+    public void onBlockPlacedBy(World worldIn, BlockPos pos, IBlockState state, EntityLivingBase placer, ItemStack stack)
+    {
+        /*
+        Only work on server side
+        */
+        if(worldIn.isRemote)
+        {
+            return;
+        }
+
+        TileEntity tileEntity = worldIn.getTileEntity(pos);
+        if(placer instanceof EntityPlayer && tileEntity instanceof TileEntityColonyBuilding)
+        {
+            TileEntityColonyBuilding hut = (TileEntityColonyBuilding) tileEntity;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'hut' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)